.github/workflows/refapp-3x-login.yml

```diff
@@ -1,17 +1,43 @@
 name: RefApp 3.x Login
 on:
+  push:
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
 jobs:
   build:
     runs-on: ubuntu-latest
+    services:
+      firefox:
+        image: selenium/node-firefox:3.141.59-gold
+        env:
+          HUB_HOST: hub
+          HUB_PORT: 4444
     steps:
-      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Comment:
Do we actually need to load Java here? (The other tests are actually run through Maven, but these tests are all Cypress based, right?)

Reply:
Yeah. No need at all. removed it.